packages/tlschema/src/misc/TLRichText.ts

```diff
@@ -0,0 +1,34 @@
+import { T } from '@tldraw/validate'
+
+/** @public */
+export const richTextValidator = T.object({ type: T.string, content: T.arrayOf(T.unknown) })
+
+/** @public */
+export type TLRichText = T.TypeOf<typeof richTextValidator>
+
+/** @public */
+export function convertTextToTipTapDocument(text: string): TLRichText {
+	const lines = text.split('\n')
+	const content = lines.map((line) => {
+		const sanitizedLine = line
+			.replace(/\x20/g, '\u00A0')
+			.replace(/&/g, '&amp;')
+			.replace(/</g, '&lt;')
+			.replace(/>/g, '&gt;')
```

Comment:
i don't think we need this - i actually think it's causing issues. when i copy-pasted a normal text element from current tldraw into the examples demo (to simulate migrating up) it came through with these escape sequences appearing in the text

Reply:
Ah, I see - this has become unnecessary now with the "all-in" PR, nixing the `text` property: https://github.com/tldraw/tldraw/pull/5131
It was a bug above, see https://github.com/tldraw/tldraw/pull/4895#issuecomment-2529907066

thanks! didn't see that this had regressed/became unnecessary with that new code.